### DISCUSSION TITLE
What's the best way to gracefully close the server?

### DISCUSSION THREAD

**Discussion by miapuffia** [scottoffen/grapevine#58] [Q&A] [answered]
server.Close() gives me errors, as does server.Dispose(). The only thing that seems to work is closing the program. Am I missing something?

I can post more information if that was supposed to work.

**Comment by scottoffen** [comment, marked as answer]
```csharp
server.Stop();
```

**Comment by ChristophKrause** [comment]
![image](https://user-images.githubusercontent.com/10434825/150005004-aafa8490-c713-44ef-96a8-6ac7a51d6e78.png)
Unfortunatelly this message pops up

**Comment by scottoffen** [comment]
I am unable to duplicate this using the latest release (5.0.0-rc.9). If you are still experiencing this issue, can you provide the minimum amount of sample code required to duplicate it so that I can look into it further?
